### ISSUE TITLE
Does it work on gcc13?

### ISSUE THREAD

**Issue by lednakashim** [serge-sans-paille/frozen#169]
getting header related errors like

```
/include/frozen/bits/pmh.h:59:3: error: 'uint64_t' does not name a type
   59 |   uint64_t seed;
```

**Comment by serge-sans-paille** [comment]
Hey @lednakashim. It looks like you're not using the latest version. This but got fixed in 5af40b3f9e03116aa2de4d55b53b9629e5988480

**Comment by serge-sans-paille** [comment]
see #170 for CI activated with gcc13
